src/routes/directions.js

```diff
@@ -1,9 +1,10 @@
 const express = require('express');
 const router = express.Router();
-
+require('dotenv').config()
 
 
 router.get('/directions', (req, res) => {
+  // let apiKey = process.env.GOOGLE_MAPS_API;
```

Comment:
Before PRing against `dev`, remove any comments that are not intended to be helpful messages for future developers. Thanks!